.travis.yml

```diff
@@ -4,17 +4,16 @@ services:
 script:
 - docker build --target test --tag my-test-image .
 - docker run my-test-image todo_app/items_test.py    
-- docker run my-test-image todo_app/integration_test.py  
```

Comment:
Now you've got this test working, it would be good to add it back to your pipeline!

Reply:
Updated, thank you